backend/apps/incidences/serializers.py

```
@@ -8,73 +8,58 @@ class Meta:
 
     class Meta:
         model = Incidence
         fields = [
             'id', 'title', 'description', 'location_type', 'room_number', 
-            'status', 'priority', 'updates', 'admin_notes', 'img', 'created_at', 'is_mine',
-            'assigned_staff_name', 
-            'assigned_staff_job',
-            'assigned_external_name'
+            'status', 'priority', 'updates', 'admin_notes', 'img', 'created_at', 'is_mine', 'student_name', 'assigned_staff',
+            'assigned_staff_name', 'assigned_staff_job', 'assigned_external_name'
         ]
```

Ruff (0.15.6)

[warning] Mutable default value for class attribute

(RUF012)